### ISSUE TITLE
机器学习资源汇总

### ISSUE THREAD

**Issue by shniu** [shniu/ai-cells#11]
#13 这里也有一部分，可以合到一起

## scikit-learn

[sklearn中文文档](http://sklearn.lzjqsdd.com/)

[awesome-python](https://awesome-python.com/)

[orange Data Mining](https://orange.biolab.si/download/)

**Comment by shniu** [comment]
## 各种大赛 题目及答案

[用户贷款风险预测（算法竞赛）](https://github.com/hczheng/Rong360)

[金陵书生大数据竞赛专栏](https://zhuanlan.zhihu.com/jlbookworm)

[https://github.com/wepe?tab=repositories](https://github.com/wepe?tab=repositories)

https://github.com/duxuhao/rong360-season2/blob/master/cadv-du-da-答辩.pdf


### 数据集
[很多数据集](https://archive.ics.uci.edu/ml/machine-learning-databases/iris/)

[uci数据集](http://archive.ics.uci.edu/ml/index.php)

**Comment by shniu** [comment]
[XGBoost + LR 就是加特征而已](http://www.algorithmdog.com/xgboost-lr-more-feas)
[CTR预估中GBDT与LR融合方案](http://blog.csdn.net/lilyth_lilyth/article/details/48032119)
[【实践】CTR中xgboost/gbdt +lr](http://blog.csdn.net/dengxing1234/article/details/73739836)


[适合入门的8个趣味机器学习项目](http://www.ppvke.com/Blog/archives/49143)

**Comment by shniu** [comment]
### 优质资源

[Gradient Boosting explained ](http://arogozhnikov.github.io/2016/06/24/gradient_boosting_explained.html)

[可视化的神经网络](https://arogozhnikov.github.io/3d_nn/)

[ROC曲线的解释](http://arogozhnikov.github.io/2015/10/05/roc-curve.html)

https://elitedatascience.com/machine-learning-interview-questions-answers

**Comment by shniu** [comment]
### 特征工程相关

[Discover Feature Engineering, How to Engineer Features and How to Get Good at It](https://machinelearningmastery.com/discover-feature-engineering-how-to-engineer-features-and-how-to-get-good-at-it/)

**Comment by shniu** [comment]
### 一步步实现神经网络算法

http://www.wildml.com/2015/09/implementing-a-neural-network-from-scratch/

http://iamtrask.github.io/2015/07/12/basic-python-network/

http://neuralnetworksanddeeplearning.com/chap1.html
